website/docs/how-to/how-to-import-export.md

```diff
@@ -31,6 +31,12 @@ Be careful when using the `drop` parameter in production environments: cleaning
 
 ### State Service {#state-service}
 
+:::caution Removal notice
+
+State Service has been removed as of Unleash 6.0
```

Comment:
Is this good enough? Should we point out the fact that Pro will "get this" before v6?